test/reuse/util/userSettings/getDateFormatFromUserSettings.spec.js

```diff
@@ -0,0 +1,24 @@
+const data = require("./data/data.json");
+
+describe("userSettings", function () {
+  const DATE_FORMAT_LENGTH = 10;
+
+  describe("getDateFormatFromUserSettings.spec - passing date format", function () {
+
+    it("Preparation: Set systemUrl ", async function () {
+      browser.config.params.systemUrl = util.data.decrypt(data.systemUrl);
+    });
+
+    it("Execution & Verification: Set User Date", async function () {
+      const userDateFormat = await util.userSettings.getDateFormatFromUserSettings(util.data.decrypt(data.username), util.data.decrypt(data.password));
+      const date = await common.date.getToday("yyyy/mm/dd");
+      const userDate = await common.date.getToday(userDateFormat);
+      //Ensure returned date matches 10 digits
+      common.assertion.expectEqual(date.length, DATE_FORMAT_LENGTH);
```

Comment:
instead of checking for the length (which might not be working for specific other configs), I would check if both dates are of the same type (DateFormatsType) since the userDateFormat could be different from time to time while the "yyyy/mm/dd" is static

Reply:
as discussed in the meeting, type cant be checked for now as we dont use ts, therefore deleted the length check and kept the date verification